manage/sawtooth_manage/subproc.py

```diff
@@ -103,13 +103,14 @@ def start(self, node_args):
             if cmd == 'validator':
                 component = '--component-endpoint', url
                 network = '--network-endpoint', gossip_port
+                public_uri = '--public-uri', gossip_port
```

Comment:
If the purpose of this is a number of processes running on the same machine, then --public-uri should probably be something like "tcp://localhost:{}".format(gossip_port_num)